### DISCUSSION TITLE
Any consumer grade hardware for running 405b? i know it's going to be slow but i can accept 0.3token/s... maybe a bit lower

### DISCUSSION THREAD

**Discussion by kolinfluence** [ggml-org/llama.cpp#8703] [Q&A]
does anyone know what hardware to get to run 405b in pure ram with maybe a bit of gpu

**Comment by dspasyuk** [comment]
@kolinfluence I wish there would be a cheap solution at this point but I do not see one. P40 GPU server would be the cheapest and the best at this point. You will need Xeon cpu with all 4 PCI slots at 16x on MBR, this will give you 96VRAM for about ~1200 USD.
